lib/datastore/request.js

```diff
@@ -148,6 +148,15 @@ DatastoreRequest.prototype.get = function(keys, callback) {
 };
 
 /**
+ * Maps to {module:datastore/dataset#save}, forcing the method to be `insert`.
+ */
+DatastoreRequest.prototype.insert = function(entities, callback) {
+  var isArray = util.is(entities, 'array');
+  entities = util.arrayize(entities).map(util.propAssign('method', 'insert'));
+  this.save(isArray ? entities : entities[0], callback);
```

Comment:
`isMultipleRequest` is used to determine if the user should get one value returned in the callback or an array.

e.g. `function callback(err, result) {}` vs `function callback(err, results) {}`